nixos/modules/misc/ids.nix

```diff
@@ -294,6 +294,7 @@
       jackett = 276;
       aria2 = 277;
       clickhouse = 278;
+      passopolis = 279;
```

Comment:
As it seems to not to persist data to disk, it does not require a fixed uid. If that's the case, just remove this line. The uid will be then allocated dynamically.